generators/bdk/python/templates/python/main.py.ejs

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import logging
+import logging.config
+import os
+
+from symphony.bdk.core.config.loader import BdkConfigLoader
+from symphony.bdk.core.service.datafeed.real_time_event_listener import RealTimeEventListener
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+from symphony.bdk.gen.agent_model.v4_initiator import V4Initiator
+from symphony.bdk.gen.agent_model.v4_message_sent import V4MessageSent
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+async def run():
+  config = BdkConfigLoader.load_from_file(os.path.join(current_dir, '..', 'resources/config.yaml'))
```

Comment:
what about putting it in the same folder? (no resources/) 
as we have just a few file for a python bot

Reply:
and do we need the python/ folder too?

Reply:
I just want to keep the same structure as the legacy one